src/Components/Square.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import {SquareProps} from '../Types/TypeProps'
+
+/**
+ * Render board cell buttons
+ * @param {*} props (Action properties from parent(<Game />) component)
+ */
+const Square: React.FC<SquareProps> = (props: SquareProps) => {
+
+  /**
+   * Handle action for each cell when clicked
+   * (This method was basically created to circumvent issue with passing down parameters)
+   * @param event 
+   */
+  const extendHandleClick = (event:any) => {
+    props.handleClick(event)
+  }
+  
+  return (
+    <button className={props.isWinnerCell ? "square"  : ""} onClick={extendHandleClick.bind(this)}>
```

Comment:
```suggestion
    <button className={props.isWinnerCell ? "square"  : ""} onClick={extendHandleClick}>
```

no need to bind, functions don't have a `this` reference set